pyproject.toml

```diff
@@ -67,6 +63,13 @@ distributed = {version = "*", optional = true}
 fsspec = {version = ">=0.3.3", optional = true}
 zarr = {version = "*", optional = true}
 
+## Pip dependencies from Git repositories
+#----------------------------------------
+geoh5py = {version = ">=0.9.1, <0.11", allow-prereleases = true}
```

Comment:
We should probably lock it on develop as well to make sure we test with the latest